apps/opik-backend/src/main/java/com/comet/opik/api/resources/v1/priv/FeedbackDefinitionResource.java

```diff
@@ -117,13 +144,12 @@ public Response update(final @PathParam("id") UUID id,
     })
     public Response deleteById(@PathParam("id") UUID id) {
 
-        var workspace = service.getWorkspaceId(id);
-
-        if (workspace == null) {
```

Comment:
Functional change here, looks fine but be careful.

Reply:
There was a leftover from when we changed to filter instead of controller level auth.